src/sysctl.rs

```diff
@@ -127,7 +128,7 @@ impl<T: SysctlStorage> Sysctl<T> {
                 extra1: ptr::null_mut(),
                 extra2: ptr::null_mut(),
             },
-            unsafe { mem::zeroed() },
+            unsafe { mem::zeroed() }
```

Comment:
OOC is this required or is this just an automatic cleanup?

Reply:
It's required, see the bug I filed.